label-application-rule-config-service-api/src/main/proto/org/hypertrace/label/application/rule/config/service/v1/label_application_rule.proto

```diff
@@ -70,18 +70,23 @@ message LabelApplicationRuleData {
   }
 
   message Action {
-    string entity_type = 1; // captures the entity type - "API", "Service"
+    repeated string entity_types = 1; // captures the entity type - "API", "Service"
     Operation operation = 2;
     oneof value {
-      string static_label_id = 3;
-      DynamicLabel dynamic_label = 4;
+      StaticLabels static_labels = 3;
+      string dynamic_label_key = 4;
+      DynamicLabel dynamic_label = 5;
```

Comment:
nit dynamic_label_expression perhaps? To disambiguate between the simpler form on 77.